pipelines/pipedrive/custom_fields_munger.py

```diff
@@ -0,0 +1,56 @@
+from dlt.common.normalizers.names.snake_case import normalize_column_name
+from typing import Dict, List, Generator, Optional
+
+import dlt
+
+
+def munge_push_func(data: List, endpoint: str) -> List:
```

Comment:
if you want your pipeline to be fully typed please add `py.typed` file as explained in new README and run `make lint`. If you use generic types like `List` or `Dict` you should parametrize them

Reply:
I'll take a look.